### PR TITLE
done

### PR DESCRIPTION
**Pull request by summyalena**
### I have implemented the tasks given below:
- Create a class Classroom with the following:
@label instance variable, should be initialized in the constructor.
- Setter and getter for @label (remember about attr_accessor).
- Create the has-many/belongs-to relationship between Classroom and Student. The following should be implemented:
- Create the has-many side (a classroom has many students).
- Create the belongs-to side (a student belongs to a classroom).
- Make sure that when adding a student to a classroom it also sets the classroom for the student.
- Make sure that when setting the classroom for a student it also adds it to the classrooms' students.
- Create a class Book with the following:
@title and @author instance variables, should be initialized in the constructor.
- Setters and getters for instance variables (remember about attr_accessor).
- Create a class Rental with the following:
@date instance variable, should be initialized in the constructor.
- Setter and getter for @date (remember about attr_accessor).
- Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
- Create the has-many side of Book and Rental (a book has many rentals).
- Create the belongs-to side of Rental and Book (a rental belongs to a book).
- Create the has-many side of Person and Rental (a person has many rentals).
- Create the belongs-to side of Rental and Person (a rental belongs to a person).
- Modify the constructor of Rental so Book and Person are set in it.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```
Comment (Deepakdanger) [inline classroom.rb]: -[ ] `student` should be in aar_reader.
For ex;-
`attr_accessor :label`
`attr_reader :students`

File: person.rb [hunk 2]
```diff
@@ -22,6 +24,10 @@ def correct_name
 
   private
 
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
```
Comment (Deepakdanger) [inline person.rb:27]: - [ ] Kindly move this function above the `private` line 25. as this is used as a public function.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @summyalena , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] README file should be descriptive
- all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo
- Fill all the  necessary details properly in correct format.
-  [ ] Kindly give good PR. as it describes the work done in this milestone.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
-Getting Started
-Clone this repo https://github.com/DuaneDave/School-lib.git
 
-git clone https://github.com/DuaneDave/School-lib.git
-Navigate to morse-code folder/directory
+# Getting Started
+--- To clone the github repo: git clone https://github.com/summyalena/school-lib/
 
 cd school-lib
 On the command line, at the project's root, run bundle install to install app dependencies
```
Comment (t-yanick) [inline README.md:12]: - [OPTIONAL] As requested by the previous code reviewer, which is a requirement for this project, please make sure you update your readme file to reflect the changes for this milestone, please checkout this [template ](https://github.com/microverseinc/readme-template)from Microverse and adapt to your project.

**Review by t-yanick — APPROVED** [review]
Hi @summyalena, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code
 
### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**